examples/basicExampleV4.php

```diff
@@ -0,0 +1,213 @@
+<?php
```

Comment:
This file will be removed after CR - it's only for testing purposes, to check that authorization flow work in the same manner on v3 and v4